public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

```

Comment:
there were a few random character at the end of this line which were rendering to the page, I simply deleted them

Reply:
Thanks for finding that dude! haha